kyo-scheduler/jvm/src/main/scala/kyo/scheduler/package.scala

```diff
@@ -0,0 +1,25 @@
+package kyo
+
+import java.io.PrintWriter
+import java.io.StringWriter
+import java.util.logging.Logger
+
+package object scheduler:
+
+    private val log = Logger.getLogger("kyo.scheduler")
+
+    private def statsScope(path: String*) =
+        "kyo" :: "scheduler" :: path.toList
+
+    private def bug(msg: String, ex: Throwable) =
+        log.severe(s"🙈 !!Kyo Scheduler Bug!! $msg \n Caused by: ${stackTrace(ex)}")
```

Comment:
Just curious, why not use the overload of this method where you can pass the `Throwable` as an argument?

I was hoping that the logging backend will do the printing of the Throwable's stack trace for us, instead of us having to do it manually with out own ad hoc method (`private def stackTrace(ex: Throwable)`).

Supplying an intact Throwable instance to the logger will also have better interop with 3rd parties. (You can always turn a Throwable to String, but you can't go the other way around.)